app/code/core/Mage/Core/Controller/Response/Http.php

```diff
@@ -99,4 +99,14 @@ public function sendHeadersAndExit(): never
         $this->sendHeaders();
         exit;
     }
+
+    /**
+     * Prepare JSON formatted data for response to client
+     */
+    public function setBodyJson($response): self
```

Comment:
```suggestion
    public function setBodyJson($response): self
```

can $response have a type?

Reply:
You're right, but it should be `mixed` since that's what the core helper method accepts:

https://github.com/MahoCommerce/maho/blob/119aeb38714d3a0fadb6aece9c2c8118ec6dd697/app/code/core/Mage/Core/Helper/Data.php#L685-L693

We could choose a more restrictive type, but it should at least include array, object, varien object, and null. But it's also valid to encode strings, numbers, and probably other types too, so it's basically just mixed anyway.

Reply:
let's go with mixed :-)

Reply:
I actually hadn't realized that mixed was a valid type in PHP8 until now. Will update later today, unless you want to do.

I've been testing the PR in various places and it works well.

Reply:
done, and if it's all green I'll merge it right away